### DISCUSSION TITLE
Documentation - Supported Actions Per Browser

### DISCUSSION THREAD

**Discussion by zero77** [Cookie-AutoDelete/Cookie-AutoDelete#949] [Ideas / Feature Requests]
**Please describe what you need support with.** 

Could you please create some kind of documentation to show what's currently supported for each browser.
Ideally with references to relevant issue or browser bug report.

**Describe the solution you'd like** 

Perhaps a table in the readme.

**Additional context** 

If it's of help, i can create the table and add the actions, if someone can add the status and references.

**Comment by kennethtran93** [comment]
Feel free to create a table either here or fork your own version, update the readme accordingly, and submit a pull request. We can see about statuses and whatnot from then on. 

May fall into the FAQ section but we'll see.

**Comment by kennethtran93** [comment]
This issue has been converted to a 'discussion' as part of an initial test of GitHub Discussions feature.

**Comment by zero77** [comment]
Related PR:
https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/pull/971
